### DISCUSSION TITLE
I keep getting warning when I use npm

### DISCUSSION THREAD

**Discussion by Eyuel-Nigussie** [quasarframework/quasar#15436] [CLI - PWA mode] [answered]
![Bad Engine](https://user-images.githubusercontent.com/105924760/218422680-4b24d539-91ab-42c7-af7b-fceef39d5a83.png)

**Comment by MilosPaunovic** [comment]
Have you been running the command with the double quotes? If so, you need to run it without it - `npm install pinia`

**Reply by Eyuel-Nigussie** [reply]
Ofcourse not, I run it as 
$ npm install pinia

Also, this is not the first occurrence of the warning error. It also shows when i run 
$ quasar ext add @quasar/testing-e2e-cypress

**Comment by pdanpdan** [comment, marked as answer]
please do not use unstable versions of node (odd major numbers)

**Reply by Eyuel-Nigussie** [reply]
This might be the answer. Let me check it out and if that's valid then I'll mark it as the answer
